### PR TITLE
Fix the stuck in fullscreen bug

### PR DESCRIPTION
**Pull request by adityaruplaha**
Direct port of #3611 from Citra.
Original PR link: https://github.com/citra-emu/citra/pull/3611

### CODE REVIEW

File: src/yuzu/main.cpp [hunk 1]
```diff
@@ -1,4 +1,5 @@
 // Copyright 2014 Citra Emulator Project
+// Copyright 2014 Citra Emulator Project
```
Comment (N00byKing) [inline src/yuzu/main.cpp]: Double Copyright header?

**Comment by adityaruplaha** [comment]
Gimme a microsecond to fix it.

File: src/yuzu/main.cpp (hunk 1 above)
Comment (daniellimws) [reply src/yuzu/main.cpp]: wanna change it to Yuzu?

Comment (N00byKing) [reply src/yuzu/main.cpp]: As both Projects share codebases, the Copyright header is only changed when major differences occur IIRC.

**Pull request merged**